site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'A new home for the Project Fugu API Showcase'
+subhead: >
+  The Project Fugu API Showcase is a collection of apps that make use of APIs that are part  of Project Fugu.
+date: 2023-03-06
+# updated: 2023-03-06
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg
+alt: Blowfish swarm swimming in the ocean.
+authors:
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+The cross-company Capabilities Project (code name [Project Fugu](/capabilities/)) at Google has the objective of making it possible for web apps to do anything platform-specific apps can. Apart from Google, project partners include Microsoft, Intel, Samsung, and others. The project enables amazing web applications like [Photoshop](https://web.dev/ps-on-the-web/) by exposing the capabilities of the underlying operating systems to the web platform, while maintaining user security, privacy, trust, and other core tenets of the web.
+
+But what are examples of apps that make use of these capabilities? Our answer to this question is the [Project Fugu API Showcase](/fugu-showcase/). It is sourced by community submissions, and contains a filterable list of apps that make use of one or more of the Fugu APIs.
+
+{% Aside 'success' %}
```

Comment:
these two seem a bit odd as "success" as that doesn't really describe their contents. I think they could just be regular asides.

Reply:
Fixed.